app/components/JobStatusCard.tsx

```diff
@@ -39,6 +41,7 @@ const JobStatusCard = ({
 }) => {
   const [versionInfo, setVersionInfo] = useState<any>(null);
   const [loadingVersion, setLoadingVersion] = useState(false);
+  const [showCosts, setShowCosts] = useState(false);
```

Comment:
would use the following hook to have it persist in local storage

https://usehooks-ts.com/react-hook/use-local-storage